### PR TITLE
Translate latest security post (ru)

### PR DESCRIPTION
**Pull request by BadAllOff**
Translate latest security post (ru)

### CODE REVIEW

**Comment by stomar** [comment]
replaces #1724 (btw you could also have force pushed your branch again instead of opening a new PR).

File: ru/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Это атака, где целью является выполнение произвольных команд в операционной 
+системе сервера через уязвимое приложение.
+Этой уязвимости назначен идентификатор [CVE-2017-17405](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-17405).
+
+## Детали
+
+`Net::FTP#get`, `getbinaryfile`, `gettextfile`, `put`, `putbinaryfile`, и
+`puttextfile` используют `Kernel#open` что бы открыть локальный файл. 
+Если аргумент `localfile` начинается с символа конвейеризации `"|"`, команда 
+следующая за символом конвейеризации будет выполнено.
```
Comment (gazay) [inline ru/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md]: `будет выполненА`

File: ru/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Значение по умолчанию для `localfile` таково `File.basename(remotefile)`, 
+`File.basename(remotefile)`, поэтому вредоносные FTP-серверы могут вызвать
```
Comment (gazay) [inline ru/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md]: You wrote default value twice here. I think better wording is:
```
Для `localfile` значением по умолчанию является `File.basename(remotefile)`...
```

**Comment by gazay** [comment]
Thank you!

**Pull request merged**